src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Telemetry/ProjectTelemetryEvent.cs

```diff
@@ -12,15 +14,32 @@ public class ProjectTelemetryEvent : TelemetryEvent
             string nuGetVersion,
             string projectId,
             NuGetProjectType nuGetProjectType,
-            bool isPRUpgradable) :
+            bool isPRUpgradable,
+            string fullPath) :
             base(ProjectInformationEventName, new Dictionary<string, object>
                 {
                     { nameof(NuGetProjectType), nuGetProjectType },
                     { nameof(NuGetVersion), nuGetVersion },
-                    { nameof(ProjectId), projectId.ToString() },
-                    { IsPRUpgradable, isPRUpgradable }
+                    { nameof(ProjectId), projectId },
+                    { IsPRUpgradable, isPRUpgradable },
                 })
```

Comment:
If you're doing nameof(xxxx) then should we follow same pattern for `IsPRUpgradable` too?